src/lib/data/cox.ts

```diff
@@ -347,6 +347,12 @@ const itemBoosts = [
 			item: getOSItem('Bandos godsword (or)'),
 			boost: 2.5
 		}
+	],
+	[
+		{
+			item: getOSItem('Dragon hunter lance'),
```

Comment:
Applying the boost like this results in a 16.7% speed upgrade as supposed the likely intended 5%. For cox to allow more boost, the boost system should be rewritten. 
As many things in slayer won't be finished before it is being released. I would propose to exclude this from the release of slayer as well, until a better cox system has been made